climada/hazard/io.py

```diff
@@ -1038,6 +1039,128 @@ def write_hdf5(self, file_name, todense=False):
                         )
         self.centroids.write_hdf5(file_name, mode='a')
 
+    def write_raster_local_exceedance_inten(self, return_periods, filename):
+        """
+        Generates exceedance intensity data for specified return periods and 
+        saves it into a GeoTIFF file.
+    
+        Parameters
+        ----------
+        return_periods : np.array or list
+            Array or list of return periods (in years) for which to calculate 
+            and store exceedance intensities.
+        filename : str
+            Path and name of the file to write in tif format.
+        """
+        inten_stats = self.local_exceedance_inten(return_periods=return_periods)
+        num_bands = inten_stats.shape[0]
+        
+        if not self.centroids.get_meta():
+            raise ValueError("centroids.get_meta() is required but not set.")
```

Comment:
What is this line for? `centroids.get_meta()` always returns the meta. Please see the updated centroids methods on develop.